### ISSUE TITLE
Upgrade to Bagisto 2.x

### ISSUE THREAD

**Issue by phpfour** [mmrtonmoybd/Bagisto-Sslcommerz-Payment-Gateway#2]
Hey,

Thanks for contributing this module. We are planning to use it for an upcoming project. Do you have any plan to make this compatible with the latest Bagisto version? 

Thanks!

**Comment by mmrtonmoybd** [comment]
> Hey,
> 
> Thanks for contributing this module. We are planning to use it for an upcoming project. Do you have any plan to make this compatible with the latest Bagisto version?
> 
> Thanks!

@phpfour 
Yes, within 2/3 days I will upgrade it and then it will be compatible with the latest Bagisto version. 

You can also use Aimeso instead of Bagisto. 
Aimeso is 5x better then Bagisto. 

You can use my another package [Aimeos Bangladeshi payment extension](https://github.com/mmrtonmoybd/bd-payments) which currently support AamarPay payment gateway soon I will include Sslcommerz

**Comment by phpfour** [comment]
Many thanks for your quickly reply, appreciate it.

I will look into Aimeos soon, it's always valuable to know from someone with experience about what's good 🙂

**Comment by mmrtonmoybd** [comment]
> Hey,
> 
> Thanks for contributing this module. We are planning to use it for an upcoming project. Do you have any plan to make this compatible with the latest Bagisto version?
> 
> Thanks!

Check out the new version [**v1.3**](https://github.com/mmrtonmoybd/Bagisto-Sslcommerz-Payment-Gateway/releases/tag/v1.3)

I hope this will work well. 
If it successfully works, then please reply in this section after your reply I close this issue.
If it has any error or bug, then create an issue also with briefly describe. 

I am waiting for your reply. 

@phpfour

**Comment by mmrtonmoybd** [comment]
successfully run with latest Bagisto version
